### PR TITLE
address readable_granule_name implementation error

### PR DESCRIPTION
**Pull request by JessicaS11**
UPDATE: After some further feedback from NSIDC and debugging, it looks like the issue was that CMR can handle a list of potential `readable_granule_name` values, while the NSIDC subsetter (EGI) can only handle one `readable_granule_name` value per submission. Thus, the warning/not implemented error messages have been removed, and icepyx instead checks for a list of values in the `readable_granule_name` parameter and submits each one as a separate order.

ORIGINAL TEXT: Addresses #278. The `readable_granule_name` keyword available in CMR is NOT implemented in the NSIDC EGI (subsetter), which processes all data orders. This is the mechanism used to search for ICESat-2 data by track and cycle. Thus, while granules can be identified via this keyword in CMR, they cannot be directly ordered + downloaded. NSIDC has issued a ticket to add this functionality; in the meantime, this PR lets users know about this limitation by adding a warning to the tutorial and issuing a not implemented error if users try to order data this way from NSIDC.

### CODE REVIEW

**Comment by JessicaS11** [comment]
@andypbarrett Are you available to review this PR? Thanks!

**Comment by JessicaS11** [comment]
> @andypbarrett Are you available to review this PR? Thanks!

or @tsutterley @trey-stafford @mikala-nsidc?

File: icepyx/core/query.py [hunk 1]
```diff
@@ -987,15 +987,38 @@ def order_granules(self, verbose=False, subset=True, email=False, **kwargs):
...
+        if "readable_granule_name[]" in self.CMRparams.keys():
+            gran_name_list = self.CMRparams["readable_granule_name[]"]
+            tempCMRparams = self.CMRparams
+            if len(gran_name_list) > 1:
+                print(
+                    "NSIDC only allows ordering of one granule by name at a time; your orders will be placed accordingly."
+                )
+            for gran in gran_name_list:
+                tempCMRparams.update({"readable_granule_name[]": gran})
+                print(tempCMRparams)
```
Comment (nitin-ravinder) [inline icepyx/core/query.py:1001]:
```suggestion

```

**Review by nitin-ravinder — APPROVED** [review]
This is good to go!

**Comment by JessicaS11** [comment]
@all-contributors add @nitin-ravinder for bug, review

**Pull request merged**